src/decays/decay_functions.cpp

```diff
@@ -325,4 +325,27 @@ double sm_down_quark_masses(softsusy::QedQcd const& qedqcd, int n)
          throw std::runtime_error("Unknown quark mass");
    }
 }
+
+double Delta_t(double z) noexcept {
+   const double lnz = std::log(z);
+   const double ln1mz = std::log(1.-z);
```

Comment:
It would be better to write `std::log1p(-z)` here. (This also applies to the next two functions.)